.env.example

```diff
@@ -8,4 +8,5 @@ REACT_APP_EOS_BLOCK_EXPLORER_URL=https://jungle3.bloks.io
 REACT_APP_VOTE_CONTRACT=wevotethefee
 REACT_APP_EVODEX_CONTRACT=evolutiondex
 REACT_APP_EVODEX_API_URL=https://jungle.api.evolutiondex.io
+REACT_APP_EVODEX_API_URL_FAILOVER=https://jungle.api-2.evolutiondex.io
```

Comment:
@adriexnet  can we add a fail over endpoint for the nodeos endpoint also : 


`https://api.eosio.cr` and `https://api.eosargentina.io`

Reply:
Well UAL dont have an interceptor that I can use to catch the error and then retry the request but I think that I can check the nodeos endpoint before init the UAL provider, but in case that endpoint fails after inited the UAL provider the failover will not work

Reply:
I see, let's address that in a separate issue then.


I think it's important since we already  had an issue earlier where a nodeos api endpoint was down , and had to build the site again just to update the en var with a new variable.